config/config.exs

```diff
@@ -36,6 +36,11 @@ config :companies,
     ]
   }
 
+config :companies, :jobs_url_checker,
+  enabled: true,
+  # 3 hours
+  interval: 10_800
```

Comment:
I realistically I think we could make this much higher, closer to 12 hours, maybe even a whole day. We don't need to update the listings in real-time.

Reply:
Sounds good to me 👍